test/conversion_test.go

```diff
@@ -512,21 +476,26 @@ spec:
   params:
   - name: STRING_LENGTH
     value: 1
-    type: string
   serviceAccountName: default
   workspaces:
-  - name: password-vault
-    secret:
-      secretName: secret-password
+  - name: empty-dir
+    emptyDir: {}
```

Comment:
can we just remove the workspaces? Or it is necessary

Reply:
yeah I think the purpose of conversion_test is to test as many non-deprecated fields as possible though this workspace does not serve a very meaningful purpose in the TaskRun/PipelineRun itself